libdnf/sack/query.cpp

```diff
@@ -925,7 +935,7 @@ int
 Query::addFilter(int keyname, int cmp_type, const char **matches)
 {
     if (keyname == HY_PKG_NEVRA_STRICT) {
-        if (!(cmp_type & HY_EQ))
+        if (!(cmp_type & HY_EQ || cmp_type & HY_GT || cmp_type & HY_LT))
```

Comment:
Please fix the description. There is a statement: "@param cmpType p_cmpType: Allowed compare types - **only HY_EQ or HY_NEQ**"